### DISCUSSION TITLE
Extending battery longevity

### DISCUSSION THREAD

**Discussion by toyssamurai** [master-hax/pixel-backup-gang#17] [General]
It's not really an issue, but there's no other place to ask for recommendations, so ... 

I saw that you have a hub that connects to a SSD as well as an Ethernet cable. I would guess that the other USB cable goes to a power adapter? I've tried several USB-C to Ethernet adapters, but none of them work with my OG Pixel. Seeing that you have a workable setup, can you recommend a hub?

In addition, do you have any advice on extending the life of the battery? I had to replace mine almost every half-year.

**Comment by master-hax** [comment]
please see https://github.com/master-hax/pixel-backup-gang/issues/6 for the hub

your best bet is to remove the battery entirely.

**Comment by toyssamurai** [comment]
> please see #6 for the hub
> 
> your best bet is to remove the battery entirely.

And connect it to a charger directly? I thought most phones won't boot without a battery even if a charger is connected

**Comment by master-hax** [comment]
Yea. You'll have to cut the PCB off the battery. You could alternatively limit charging in software.

Battery management is currently not in scope for this repository and I cannot provide support for it. Good luck.

**Comment by xerloq** [comment]
> And connect it to a charger directly? I thought most phones won't boot without a battery even if a charger is connected

There's good info in this thread: https://www.reddit.com/r/GooglePixel/comments/z3z2wo/anyone_running_a_pixel_1_without_a_battery/

Specifically: https://www.reddit.com/r/GooglePixel/comments/z3z2wo/comment/ki2qrfw/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

Basically, remove the PCB from the battery and solder the charging leads to the proper points on the PCB and the phone is none the wiser re: the battery. Of course there's more to it (getting the correct voltage, etc.) but that's the simple version.
